.gitignore

```diff
@@ -18,4 +18,4 @@ __pycache__
 *.egg-info
 .ipynb_checkpoints
 src/hyperspectral/notebooks/ECOSTRESS/*
-src/hyperspectral/notebooks/data/*
\ No newline at end of file
+src/hyperspectral/notebooks/data/*.tar.gz
```

Comment:
The notebooks download and unpack data into the `data` directory during their execution which is not covered by the `.tar.gz` extension.  One can explicitly add files in this directory into the repo; is there a reason to remove the broader ignore pattern?

Reply:
In my case, I do not have the data directory mounted into the container.  I can see why this is inconvenient for you.  I will revert this change.

Reply:
> I've filed [CSTARS/ecosis#53](https://github.com/CSTARS/ecosis/issues/53) to check on whether we can redistribute spectrum data.

I've removed this addition.  If approved by them, it can be added in this or a subsequent PR.

Reply:
I'm not sure why the comment above appeared in this part of the thread.  In any case, I have removed the change to the `.gitignore` file.